lib/common-lib/src/components/layout/HeaderTags/BodyMedium.js

```diff
@@ -3,7 +3,7 @@ import { Text } from 'native-base'
 
 const BodyMedium = ({ children, ...props }) => {
   return (
-    <Text {...props} fontSize='14px' fontWeight='400'>
+    <Text {...props} fontSize='12px' fontWeight='400'>
```

Comment:
@reddyganesha15 are we also having something for `font-weights`? Or we can hard code it?